components/TaskContainer.vue

```diff
@@ -92,16 +114,24 @@ export default {
   },
   mounted() {
     this.$store.dispatch('todos/setTotalPage');
-    this.$store.dispatch('todos/filterTaskList', this.filterOptions[0]);
+    this.$store.dispatch('todos/setActiveFilterOption', this.filterOptions[0]);
+    this.$store.dispatch('todos/filterTaskList');
   },
   methods: {
-    showAddTodoCard() {
+    async showAddTodoCard() {
+      await this.$store.dispatch('todos/setSearchText', '');
+      this.$store.dispatch('todos/setShowSearchField', false);
```

Comment:
Why are we clearing out search fields when showing the add card? this introduces a delay after clicking the create button before showing the card.

Reply:
await removed as it's not needed here